_pkdown.yml

```diff
@@ -0,0 +1,2 @@
+template:
```

Comment:
It's fortunate that the alphabetical order of articles is the right one so we don't need to add an articles index here :grin:

Reply:
Yes, that might change, but for now we're good! 🥳